dreadnode/cli/platform/cli.py

```diff
@@ -122,43 +235,57 @@ def configure(
     env_overrides = {}
     for i in range(0, len(args), 2):
         key = args[i]
-        value = args[i + 1] if not unset else None
-        env_overrides[key] = value
+        env_overrides[key] = args[i + 1] if not unset else None
 
-    configure_platform(tag=tag, **env_overrides)
+    if not env_overrides:
+        print_warning("No configuration changes specified.")
+        return
+
+    print_info("Setting environment overrides ...")
+    build_compose_override_file(PLATFORM_SERVICES, current_version)
+    build_env_file(current_version.configure_overrides_env_file, **env_overrides)
+    print_info(
+        f"Configuration written to {current_version.local_path}.\n\n"
+        "These will take effect the next time the platform is started. "
+        "You can modify or remove them at any time."
+    )
```

Comment:
Generally did a lot of inlining of helper functions that were only used in a single place. I don't think we should be fighting to keep code "small" in arbitrary places at the trade-off having to jump around to understand functionality + more arg typing + more docstrings.